petclinic-frontend/src/features/carts/components/UserCart.tsx

```diff
@@ -43,6 +51,16 @@ const UserCart = (): JSX.Element => {
         setCartItems(products);
         const initialPrices = products.map(item => item.productSalePrice);
         setFixedPrice(initialPrices);
+        const calculatedSubtotal = products.reduce(
+          (acc, item) => acc + item.productSalePrice * item.quantity,
+          0
+        );
+        setSubtotal(calculatedSubtotal);
+        const tvqValue = calculatedSubtotal * 0.09975; // 9.975%
+        const tvcValue = calculatedSubtotal * 0.05; // 5%
```

Comment:
Consider making tax rates configurable for future-proofing.

Reply:
Some symbols are already added to the frontend, but I will think about how to make it better